Makefile

```diff
@@ -182,5 +182,6 @@ $(eval $(call build-package,zip,3.0-r0))
 $(eval $(call build-package,libpng,1.6.38-r0))
 $(eval $(call build-package,freetype,2.12.1-r0))
 $(eval $(call build-package,fontconfig,2.14.0-r0))
+$(eval $(call build-package,openjdk17,17.0.4.1-r0))
```

Comment:
lets just call it openjdk :)

Reply:
OK cool.  I wasn't sure if we needed to include the major version in the name so we could have multiple LTS version packages. Or maybe this Java package should track the latest (currently Java 19) though I suspect that may be a problem for some of the Java based applications we want to build as a lot are using 17.  

Or best to merge this package without the version in the name and figure out different major versions after?  The supported Java versions is interesting https://endoflife.date/java